sass/config/_fonts.scss

```diff
@@ -9,8 +9,14 @@ $font-formats: 'woff2' 'woff' 'ttf';
 // ------------
 /// A font imported from typekit.
 ///
+/// - call the specimen preview directly
+/// - give a font-name and parenthetical list of variants on the first line
+/// - suppy additional html head as a nested block
+///
 /// @group fonts
-/// @preview font-specimen; key: typekit; show: regular, italic, bold, bold italic
+/// @font-specimen 'typekit' (regular, italic, bold, bold italic) -
```

Comment:
@mirisuzanne I don't think SassDoc annotations can be kebab-case. Okay iff we just shorten to `@font`?

Reply:
sure!